modules/dataquery/jsx/react.filterBuilder.js

```diff
@@ -50,7 +50,9 @@ var FilterRule = React.createClass({
 				"lessThanEqual" : "<=",
 		    	"greaterThanEqual" : ">=",
 		    	"startsWith" : "startsWith",
-		    	"contains" : "contains"
+		    	"contains" : "contains",
+		    	"isNull" : "isNull",
+		    	"isNotNull" : "isNotNull"
```

Comment:
Is this what shows to the user in the front end? If so, "isNull" and "isNotNull" is probably confusing for non-programmers. Can it be something like "is not set" or "contains no value" (or ask @LeighMac for wording that makes sense.)